easybuild/framework/easyconfig/constants.py

```diff
@@ -51,4 +51,9 @@
     'OS_VERSION': (get_os_version(), "System version"),
     'SYS_PYTHON_VERSION': (platform.python_version(), "System Python version (platform.python_version())"),
     'SYSTEM': ({'name': 'system', 'version': 'system'}, "System toolchain"),
+
+    'OSPACKAGES_IBVERBS': (('libibverbs-dev', 'libibverbs-devel', 'rdma-core-devel'),
+                           "OS packages providing ibverbs support"),
+    'OSPACKAGES_OPENSSL': (('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
```

Comment:
@terjekv To avoid confusion with the `openssl` package, should we rename this constant to `OSPACKAGES_OPENSSL_DEV` or something a bit shorter, like `OS_PKG_OPENSSL_DEV`?

Maybe we should also add `OS_PKG_OPENSSL` (for `'openssl'`) and `OS_PKG_OPENSSL_LIB` (for `('libssl', 'libopenssl')`)?

Reply:
Resolved in https://github.com/easybuilders/easybuild-framework/pull/3309/commits/2f967ae6496125390d1d94bda9ecba13cccd7563